ui/lib/core/addon/components/overview-card.hbs

```diff
@@ -20,9 +20,17 @@
     {{/if}}
   </div>
 
-  <Hds::Text::Body @color="faint" data-test-overview-card-subtitle={{@cardTitle}}>
-    {{@subText}}
-  </Hds::Text::Body>
+  {{! Pass @subText for text only content to use default styling. }}
+  {{#if @subText}}
+    <Hds::Text::Body @color="faint" data-test-overview-card-subtitle={{@cardTitle}}>
+      {{@subText}}
+    </Hds::Text::Body>
+  {{/if}}
+
+  {{! Use the "subtext" yield for stylized subtext or including elements like doc links. }}
+  {{#if (has-block "subtext")}}
```

Comment:
I didn't want to update `@subText` for all `OverviewCards`, I felt like it made sense to keep both potential use cases. 1) just text subtext and 2) more stylized content, such as inline links

Reply:
Are `@subtext` and `<:subtext>` mutually exclusive? Right now passing both with yield both

Reply:
I intentionally made them not mutually exclusive as I didn't see a reason for them to be. Someone could add subtext in the default styling and then add a second stylized block. What do you think?

Reply:
Makes sense to me! I think the most confusing part is that they are named the same thing, so it felt like maybe they should be mutually exclusive. But, naming is hard 🤷